tests/test_bitbucket.py

```diff
@@ -0,0 +1,72 @@
+from arche.tools import bitbucket
+import pytest
+
+
+urls = [
+    (
+        "https://bitbucket.org/scrapinghub/customer/src/master/customer/schemas/ecommerce.json",
+        "https://api.bitbucket.org/2.0/repositories/scrapinghub/customer/src/master"
+        "/customer/schemas/ecommerce.json",
+    ),
+    (
+        "https://bitbucket.org/scrapinghub/customer/raw/"
+        "9c4b0bf46f2012ab38bc066e1ebe774d72856013/customer/schemas/ecommerce.json",
+        "https://api.bitbucket.org/2.0/repositories/scrapinghub/customer/src/"
+        "9c4b0bf46f2012ab38bc066e1ebe774d72856013/customer/schemas/ecommerce.json",
+    ),
+]
+
+
+@pytest.mark.parametrize(
+    ["url", "expected"],
+    [
```

Comment:
I think we also need to account for broken urls, what happens if I feed `https://bitbucket.org/ecommerce.json`?
And if url can be parsed and incorrect, I guess we will get 404 or something?